tests/foreman/api/test_contentview.py

```diff
@@ -278,6 +279,63 @@ def test_positive_add_sha512_rpm(self, content_view, module_org, module_target_s
             content_view_version.errata_counts['total'] == CUSTOM_RPM_SHA_512_FEED_COUNT['errata']
         )
 
+    @pytest.mark.tier2
+    def test_ccv_promote_registry_name_change(self, module_target_sat, module_sca_manifest_org):
+        """Testing CCV promotion scenarios where the registry_name has been changed to some
+        specific value.
+
+        :id: 41641d4a-d144-4833-869a-284624df2410
+
+        :steps:
+
+            1) Sync a RH Repo
+            2) Create a CV, add the repo and publish it
+            3) Create a CCV and add the CV to it, then publish it
```

Comment:
should be content view version
```suggestion
            3) Create a CCV and add the CV version to it, then publish it
```